src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRenta = 40;
+  let totalCost = days * dayRenta;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Use a constant for the number of days that qualify for a long term discount and the discount amount itself to avoid hardcoding values directly in the expression.